starport/services/scaffolder/module.go

```diff
@@ -168,6 +168,11 @@ func (s *Scaffolder) CreateModule(moduleName string, options ...ModuleCreationOp
 
 // ImportModule imports specified module with name to the scaffolded app.
 func (s *Scaffolder) ImportModule(name string) error {
+	// Only wasm is currently supported
+	if name != "wasm" {
+		return errors.New("module cannot be imported. Supported module: wasm")
```

Comment:
```suggestion
		return errors.New("module cannot be imported. Supported modules: wasm")
```

Reply:
Well, since there is only one suported module I would let it like that. We will need to modify this line when adding new modules anyway